Update/omot_014.txt

```diff
@@ -1665,14 +1665,14 @@ void main()
 //r話を聞いてほっとすると同時に、俺は心底満たされた思いだった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "話を聞いてほっとすると同時に、俺は心底満たされた思いだった。",
-			NULL, "I felt deeply relieved to hear that things worked out.", Line_Normal);
+			NULL, "I felt relieved and deeply satisfied to hear that things worked out.", Line_Normal);
 	ClearMessage();
 
 //詩音rvS14/06/210600538.「んで、せっかくだからってことで…実はついさっきまで、一緒だったんですよ。kvS14/06/210600539.二人して学校サボって、どーでもいい話で盛り上がっていました。kvS14/06/210600540.あっはっはっは…！」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600538", 256, TRUE);
 	OutputLine(NULL, "「んで、せっかくだからってことで…実はついさっきまで、一緒だったんですよ。",
-			NULL, "\"Well, it was a really stressful experience, so... we were actually still together until just a moment ago.", Line_WaitForInput);
+			NULL, "\"Well, we got an opportunity to be together, so... we were actually still together until just a moment ago.", Line_WaitForInput);
```

Comment:
```suggestion
			NULL, "\"Well, since we had the opportunity to be together... we were actually still together until just a moment ago.", Line_WaitForInput);
```